corporate.html

```diff
@@ -205,7 +205,13 @@ <h2 style="text-align: center">On behalf of University of Nebraska Dance Maratho
           Hospitals, we would like to express our appreciation for your consideration of our cause. We are looking
           forward to working with you and your business in the future!</h2>
       </div>
-    </div>
+  </div>
+
+
+  <div class="row" style="border-bottom: 2px solid #D3D3D3;">
+    <h2><br>HuskerThon 2020 Sponsors</h2>
+  </div>
+
 
```

Comment:
Can you take out the hyperlinks for all the sponsor pages? Joe said we only want the 2021 sponsors to have hyperlinks!

Reply:
yes! I just added a commit called "remove hyperlinks from 2020 sponsors".